### PR TITLE
Deliver GenerateFHIRFromMimic Nifi Processor and consolidate health patterns NAR

### PR DESCRIPTION
**Pull request by ibmatc**
This request defines the GenerateFHIRFromMimic Nifi processor, which will produce FHIR bundles for each patient found in a supplied Mimic DB.  Also, all custom Nifi processors for Health Patterns will now be built into a single NAR, rather than separate NARs for each processor.

### CODE REVIEW

File: clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml [hunk 1]
```diff
@@ -39,14 +39,14 @@ fhir:
   env:
     # Per the FHIR team this is recommended to avoid some time out issues seen when deploying recently.
     - name: FHIR_TRANSACTION_MANAGER_TIMEOUT
-      value: '300s'
+      value: '600s'
```
Comment (luisgarcc) [inline clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml:42]: Should we create a github task to look into investigating expanding the pod's resources (i.e. mem) for the Mimic scenario?

File: nifi-processors/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM busybox
+COPY nifi-health-patterns-bundle/nifi-health-patterns-nar/target/nifi-health-patterns-nar*.nar /
```
Comment (luisgarcc) [inline nifi-processors/Dockerfile:2]: Now that the NARs are consolidated, it's great to have this here instead

File: nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/mimic/GenerateFHIRFromMimic.java [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            final byte[] data = nextBundle.getBytes();
+            flowFile = session.write(flowFile, new OutputStreamCallback() {
+                @Override
+                public void process(final OutputStream out) throws IOException {
+                    out.write(data);
+                }
+            });
+            session.getProvenanceReporter().create(flowFile);
+            session.transfer(flowFile, SUCCESS);
+        } catch (Throwable t) {
```
Comment (luisgarcc) [inline nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/mimic/GenerateFHIRFromMimic.java:209]: Shouldn't this path redirect to some failed relationship?
```
session.transfer(flowFile, FAIL_RELATIONSHIP);
```

**Review by luisgarcc — APPROVED** [review]
LGTM

File: clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml (hunk 1 above)
Comment (ibmatc) [reply clinical-ingestion/helm-charts/alvearie-ingestion/values.yaml:42]: Added https://trello.com/c/0nwh644C/200-reconsider-timeouts-calling-fhir-server-for-validation

File: nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/mimic/GenerateFHIRFromMimic.java (hunk 3 above)
Comment (ibmatc) [reply nifi-processors/nifi-health-patterns-bundle/nifi-health-patterns-processors/src/main/java/com/ibm/healthpatterns/processors/mimic/GenerateFHIRFromMimic.java:209]: I don't believe so. Unlike other processors, this is generating flowfiles from nothing. If an exception occurs, I don't think we want to produce a new flowfile with an error state. I think we just want to log a failure.

**Pull request merged**